rts/lra/lra-cdi-rest/foobar.log

```diff
@@ -0,0 +1,255 @@
+22:08:15,047 WARN  [org.jboss.as.txn] (ServerService Thread Pool -- 15) WFLYTX0013: Node identifier property is set to the default value. Please make sure it is unique.
```

Comment:
This should be removed I expect

Reply:
yes, the file should be removed. this file was added to the repo by my mistake